Lib/test/test_concurrent_futures.py

```diff
@@ -759,6 +766,180 @@ def test_ressources_gced_in_workers(self):
                                        ProcessPoolForkserverMixin,
                                        ProcessPoolSpawnMixin))
 
+def hide_process_stderr():
+    import io
+    setattr(sys, "stderr", io.StringIO())
```

Comment:
Is there a subtlety that I'm missing, or can this simply be written as `sys.stderr = io.StringIO()`?